apps/web/components/dialog/ReassignDialog.tsx

```diff
@@ -179,33 +243,70 @@ export const ReassignDialog = ({
+            isManagedEvent === null
+              ? t("loading")
+              : isManagedEvent
+                ? t("reassign_managed_event_user")
+                : t("reassign_round_robin_host")
+          }
+          description={
+            isManagedEvent === null
+              ? ""
+              : isManagedEvent
+                ? t("reassign_to_another_user")
+                : t("reassign_to_another_rr_host")
+          }
           enableOverflow>
           <Form form={form} handleSubmit={handleSubmit} ref={animationParentRef}>
             <RadioArea.Group
               onValueChange={(val) => {
                 const reassignType: ReassignType = z.nativeEnum(ReassignType).parse(val);
                 form.setValue("reassignType", reassignType);
               }}
-              defaultValue={bookingFromRoutingForm ? ReassignType.TEAM_MEMBER : ReassignType.ROUND_ROBIN}
+              defaultValue={bookingFromRoutingForm ? ReassignType.TEAM_MEMBER : ReassignType.AUTO}
               className="mt-1 flex flex-col gap-4">
               {!bookingFromRoutingForm ? (
                 <RadioArea.Item
-                  value={ReassignType.ROUND_ROBIN}
+                  value={ReassignType.AUTO}
                   className="w-full text-sm"
                   classNames={{ container: "w-full" }}
-                  disabled={bookingFromRoutingForm}>
-                  <strong className="mb-1 block">{t("round_robin")}</strong>
-                  <p>{t("round_robin_reassign_description")}</p>
+                  disabled={bookingFromRoutingForm || isManagedEvent === null}>
+                  <strong className="mb-1 block">
+                    {isManagedEvent === null
+                      ? t("loading")
+                      : isManagedEvent
+                        ? t("auto_reassign")
+                        : t("round_robin")}
+                  </strong>
+                  <p>
+                    {isManagedEvent === null
+                      ? ""
+                      : isManagedEvent
+                        ? t("auto_reassign_description")
+                        : t("round_robin_reassign_description")}
+                  </p>
                 </RadioArea.Item>
               ) : null}
               <RadioArea.Item
                 value={ReassignType.TEAM_MEMBER}
                 className="text-sm"
-                classNames={{ container: "w-full" }}>
-                <strong className="mb-1 block">{t("team_member_round_robin_reassign")}</strong>
-                <p>{t("team_member_round_robin_reassign_description")}</p>
+                classNames={{ container: "w-full" }}
+                disabled={isManagedEvent === null}>
+                <strong className="mb-1 block">
+                  {isManagedEvent === null
+                    ? t("loading")
+                    : isManagedEvent
+                      ? t("specific_team_member")
+                      : t("team_member_round_robin_reassign")}
```

Comment:
TODO: Figure out a way to not need this brief loading state, but load the dialog directly with the necessary info